projects.yaml

```diff
@@ -1250,6 +1250,14 @@ projects:
   license: ISC
   labels: [markdown]
   category: markdown
+- name: Mkdocs Quiz
+  markdown_extension: mkdocs_quizz
+  pypi_id: mkdocs-quizz
+  github_id: bdallard/mkdocs_quiz
+  license: MIT
+  labels: [markdown]
+  category: markdown
```

Comment:
But it's not a Markdown extension, right? It's a MkDocs plugin. Therefore it should be in another category, it should specify `mkdocs_plugin` instead of `markdown_extension`, and it should have a `plugin` label instead of a `markdown` one.

Reply:
Indeed thanks !